tests/loginTests.test.jsx

```diff
@@ -0,0 +1,371 @@
+  test("renders the component", () => {
+    const socialAuthButtonComponent = shallow(<SocialAuthButton />);
+    expect(socialAuthButtonComponent).toMatchSnapshot();
+  });
+});
+
+describe("<Forms />", () => {
+  test("renders the component", () => {
+    const formComponent = shallow(<Forms auth={props.auth} />);
+    expect(formComponent).toMatchSnapshot();
+  });
+});
+
+describe("<Header />", () => {
+  test("renders the component", () => {
+    const headerComponent = shallow(<Header />);
+    expect(headerComponent).toMatchSnapshot();
+  });
+});
+
+describe("<Home />", () => {
+  test("renders the component", () => {
+    const homeComponent = shallow(<Home />);
+    expect(homeComponent).toMatchSnapshot();
+  });
+});
+
+describe("<Inputs />", () => {
+  test("renders the component", () => {
+    const inputComponent = shallow(<Inputs required />);
+    expect(inputComponent).toMatchSnapshot();
+  });
+});
+
+describe("<Button />", () => {
+  test("renders the component", () => {
+    const buttonComponent = shallow(<Button options={props.options} />);
+    expect(buttonComponent).toMatchSnapshot();
+  });
+});
+
+describe("test the login container", () => {
+  let wrapper;
+  beforeEach(() => {
+    store = mockStore({
+      auth: {
+        errors: "User with this email or password was not found !"
+      },
+    });
+  });
+  it("should mount view all component", () => {
+		const loginUser = jest.fn();
+    wrapper = mount(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Login props={props} onSubmit={loginUser} />
+        </Provider>
+      </MemoryRouter>,
+    );
+    wrapper.simulate("submit");
+    expect(wrapper.find("Login").length).toBe(1);
+  });
+});
+
+
+describe("<Inputs />", () => {
+  describe("onChange()", () => {
+    test("successfully calls the onSubmit handler", () => {
+      const mockOnChange = jest.fn();
+      const wrapper = shallow(
+        <Inputs onChange={mockOnChange} label="Sign In" />
+      );
+      // const component = wrapper.dive();
+
+      wrapper.simulate("onChange");
+
+      expect(mockOnChange).toBeTruthy();
+    });
+  });
+});
+
+const initialState = {errors:"", user: null};
+describe("login reducer ", () => {
+  test("tests  log in a user", () => {
+    const token = "gdgdhdj";
+    const action = {
+      type: "LOGIN",
+      payload: token
+    };
+    expect(login({}, action)) !== initialState;
+    expect(login({email: "", password:""})).toBeTruthy();
+    expect(loginChecker({}, action)).toEqual({token});
+
+  });
+});
+
+describe("login reducer ", () => {
+  test("tests  log in a user", () => {
+    const errors = "this is an error!";
+    const action2 = {
+      type: "LOGIN_ERROR",
+      payload: errors
+    };
+    expect(login({}, action2)) !== initialState;
+    expect(login({email: "", password:""})).toBeTruthy();
+    expect(loginChecker({}, action2)).toEqual({errors});
+
+  });
+});
+
+
+// describe("login reducer", ()=>{
```

Comment:
Remove commented out code

Reply:
@shakira, working on it, the hound file must have been reintroduced when I rebates to fix unmatchable histories, going to remove it and the commented out code

Reply:
@shakirandagire I am done with requested changes